tests/test_main.py

```diff
@@ -12,19 +14,17 @@
         ("Abracadabra", "A", 5),
         ("", "a", 0),
         ("Samsung", "b", 0),
-    ]
+    ],
 )
 def test_count_occurrences(phrase, letter, count):
     assert count_occurrences(phrase, letter) == count, (
         f"Function 'count_occurrences' should return {count}, "
         f"when 'phrase'='{phrase}' and 'letter'='{letter}'"
     )
 
-    
-def test_removed_comment():
-    import app
-    with open(app.main.__file__, "r") as f:
-        file_content = f.read()
-        assert "# write your code here" not in file_content, \
-            "You have to remove the unnecessary comment '# write your code here'"
 
+def test_removed_comment():
```

Comment:
This test function 'test_removed_comment' seems to be checking if a specific comment line is removed from the function source code. This is not a usual practice. The presence of a comment does not usually affect the functionality of the code. It might be better to focus on testing the functionality of the code rather than its comments.